neural_compressor/adaptor/pytorch.py

```diff
@@ -2918,6 +2924,11 @@ def _post_hook_for_qat(self):
             PyTorch_FXAdaptor.convert_sub_graph(self.sub_module_list, \
                                                 self.model._model, prefix='')
 
+        if self.approach != 'post_training_dynamic_quant':
+            self._get_scale_zeropoint(self.model._model, self.model.q_config)
+        self._dump_model_op_stats(self.model._model, self.model.q_config, self.approach)
+        torch_utils.util.get_embedding_contiguous(self.model._model)
```

Comment:
does this make sense? it looks like NLP specific func call?

Reply:
All int8 model will execute these post-processes, won't have impact on cv models.

Reply:
this logic exists in quantize func and I just move it to hook func.